uber/site_sections/accounts.py

```diff
@@ -174,3 +186,37 @@ def sitemap(self):
                             'path': '/{}/{}'.format(module_name, name)
                         })
         return {'pages': sorted(pages.items())}
+
+    @ajax
+    def add_bulk_admin_accounts(self, session, message='', **params):
+        ids = params.get('ids')
```

Comment:
only minor suggestion and doesn't have to happen now: you may want to wrap the entire thing here in a try->except and pass the result of any exceptions back as 'message' and use that log_verbose_exception_thingy().  That way if there are any errors, the error message will come through.  (unhandled exceptions in AJAX requests come back weird)